gematria/datasets/convert_bhive_to_llvm_exegesis_input.cc

```diff
@@ -40,10 +42,60 @@ namespace {
 unsigned int file_counter = 0;
 }
 
+enum class annotator_type { exegesis, fast };
```

Comment:
Nit: This should follow the Google naming conventions with
`AnnotatorType`, `kExegesis`, and `kFast`.